packages/lesmis/src/views/DashboardView.js

```diff
@@ -197,7 +198,11 @@ export const DashboardView = React.memo(() => {
                   }}
                 />
                 {useYearSelector && (
-                  <YearSelector value={selectedYear} onChange={setSelectedYear} />
+                  <YearSelector
+                    value={selectedYear}
+                    onChange={setSelectedYear}
+                    isLoading={!!isFetching}
```

Comment:
Is the double bang necessary? Would have thought it was already a bool

Reply:
yes because the global isFetching hook returns the number of queries that are loading which is an int.